vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -466,6 +490,68 @@ and
 
         setup (siteProvider.GetProjectSite()) |> ignore
 
+    member private __.ProvideProjectSiteProvider(workspace:Workspace, project:Project) =
+        let visualStudioWorkspace = workspace :?> VisualStudioWorkspace
+        let hier = visualStudioWorkspace.GetHierarchy(project.Id)
+        {new IProvideProjectSite with
+             member this.GetProjectSite() =
+                 let compileItems () = [| for document in project.Documents do yield document.FilePath |]
+                 let compilerFlags () = [| "" |]
```

Comment:
Presumably this is the piece that we'll have to address by storing the string in the `Workspace` [here](http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/ProjectSystem/AbstractProject_Options.cs,d5006b93879b9a6e,references)